src/cpp/session/projects/SessionProjects.cpp

```diff
@@ -74,6 +74,17 @@ Error validateProjectPath(const json::JsonRpcRequest& request,
    return Success();
 }
 
+Error getWorkingDirectory(const json::JsonRpcRequest& request,
```

Comment:
You don't need this since WorkbenchContext already has a method to get the current working directory (it's kept in sync as the working dir changes): https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/workbench/WorkbenchContext.java#L116-L119